### ISSUE TITLE
Speed up multiple users sync

### ISSUE THREAD

**Issue by kocane** [croneter/PlexKodiConnect#379]
I've set up 4 profiles on our Kodi, using the "share with default" option to share the media database etc. with the master user. It works alright, but when switching users there is a 30 second delay before it's all "sync'd up" so that the homescreen sports the movies/tvshows related to the user that just logged in.

Is this expected or can should it be faster? Can something done to speed this process up, or is it related to my user setup? If I set up seperate mediainfo/mediasources for each user, would it run better? I just worry that it'll be filled with duplicate stuff, on the little raspberry pi.

**Comment by croneter** [comment]
There's 2 ways to switch users, of which you should always use the [Kodi user profiles](http://kodi.wiki/view/Profiles) and **NOT** the PlexKodiConnect menu option "Log out user XYZ"! Also see the wiki: https://github.com/croneter/PlexKodiConnect/wiki/Several-Plex-Users

I hope you've been doing the latter, which takes quite a while, because PKC needs to resync the entire Kodi library from scratch.

**Comment by kocane** [comment]
@croneter - Nah, I've only been working with Kodi profiles. I was just about to close this one myself since I went along with creating profiles that had entirely seperate mediasources and mediainfo, i.e. I had to set up PKC on each user where it also had to sync the libraries for each user. That way whenever a different user is selected, it shows the in-progress movies/shows for that user instantly and doesn't show the in-progress stats for the previous user, until it has sync'd up with plex after ca. 30 secs. I figured sharing the mediasources/mediainfo would be smart since it would only have to sync whatever is in the libraries once, but if the other method works bettr, than I'll go with that.
